chevah_build

```diff
@@ -305,7 +351,7 @@ if [ "${OS%solaris*}" = "" ]; then
         # Required for linking to libedit, which has a simpler configure setup.
         export CFLAGS="$CFLAGS -xcode=pic32"
     fi
-elif [ "${OS%freebsd*}" = "" ]; then
+elif [ "${OS%freebsd*}" = "" -o "${OS%openbsd*}" = "" ]; then
```

Comment:
maybe is best to have a separate elif for openbsd as the next comment only talks about freebsd 10

Reply:
Actually this is about Clang, which is the compiler used in both FreeBSD and OpenBSD nowadays. Will update the comment to reflect this. Thanks for spotting it!

I think after getting rid of support for OS X 10.8, it would be worth checking if changing the condition to look for `$CC = "clang"` works for MacOS too.